modules/core/test/v2/integration/wallet.ts

```diff
@@ -799,4 +799,26 @@ describe('V2 Wallet:', function() {
 
   });
 
+  describe('Algo non partipation key reg transaction', function() {
```

Comment:
This really belongs in `test/v2/integrations/coins/algo.ts`

Reply:
It has been moved in coin specific test file